docs/build/getting-started/authenticating-with-lit.md

```diff
@@ -1 +1,43 @@
-# Authenticating with Lit
\ No newline at end of file
+# Authenticating with Lit
+After connecting, sending requests to the Lit network requires authentication. This authentication is necessary to ensure that the Lit network can identify your session and only grant you access to the resources you're authorized to access.
+
+## Session Signatures
+Lit uses Session Signatures to authenticate your session. A detailed description of Session Signatures can be found [here](../session-sigs/overview.md).
```

Comment:
Consider "Authentication is handled via Session Signatures. Session signatures allow you to use a specific Lit "Ability" (i.e. signing a transaction) with a particular Lit "Resource" (i.e. a Programmable Key Pair). You can find a detailed explanation here"